mhs/outbound/outbound/request/synchronous/handler.py

```diff
@@ -26,6 +31,92 @@ def initialize(self, workflows: Dict[str, workflow.CommonWorkflow],
 
     @timing.time_request
     async def post(self):
+        """
+        ---
+        summary: Make a request to the MHS
+        description: Make a request to the MHS
+        operationId: postMHS
+        parameters:
+          - name: Interaction-Id
+            in: header
+            required: true
+            schema:
+              type: string
+            description: ID of the interaction that you want to invoke. e.g. QUPC_IN160101UK05
+          - name: sync-async
+            in: header
+            required: true
+            schema:
+              type: boolean
+            description: >-
+              If set to true and the interaction ID is for an async interaction
+              that supports sync-async, then the HTTP response will be the
+              response from Spine, and the response will not be put onto the
+              inbound queue.
+
+
+              If set to false for an async interaction, then the response from
+              Spine will be put onto the inbound queue and the HTTP response will
+              just acknowledge sending the request successfully to Spine.
+
+
+              For sync interactions or async interactions that don't support
+              sync-async, this header must be set to false.
+          - name: from-asid
+            in: header
+            required: false
+            schema:
+              type: string
+            description: from-asid
```

Comment:
@colin-bateson @Philip-Woods Do either of you know what this from-asid is (enough to write a description of it here)?

Reply:
It's the ASID of the sending system, specifically used to generate the of headers the `sync` workflow, this should match the `from-asid` value used within the hl7 body of a sync message